sources/Outputs/src/Job.cpp

```diff
@@ -138,27 +138,40 @@ Job::writeOutputs() const {
   output->setLogsEntry(log);
 
   auto final_state = job::FinalStateEntryFactory::newInstance();
-  final_state->setExportIIDMFile(exportIIDMFile_);
+
+  if (isOutputChosen("STEADYSTATE") || def_.chosenOutputs.empty()) {
+    final_state->setExportIIDMFile(exportIIDMFile_);
+    output->addFinalStateEntry(final_state);
+  }
   final_state->setExportDumpFile(exportDumpFile_);
-  output->addFinalStateEntry(final_state);
 
   auto constraints = boost::shared_ptr<job::ConstraintsEntry>(new job::ConstraintsEntry());
```

Comment:
The constraints file should be in the list. DO you agree @marcochiaramello ?

Reply:
yes, agreed!